src/crc32c.c

```diff
@@ -116,8 +117,8 @@ static uint32_t crc32c_sse42(uint32_t crc, const char *buf, size_t len)
             __asm__(CRC32_PTR "\t" "(%3), %0\n\t"
                     CRC32_PTR "\t" LONGx1 "(%3), %1\n\t"
                     CRC32_PTR "\t" LONGx2 "(%3), %2"
-                    : "=r"(crc0), "=r"(crc1), "=r"(crc2)
-                    : "r"(buf), "0"(crc0), "1"(crc1), "2"(crc2));
+                    : "+r"(crc0), "+r"(crc1), "+r"(crc2)
+                    : "r"(buf), "m"(* (const char (*)[LONG*3]) buf));
```

Comment:
I merged this suggestion by @vtjnash, but upon reflection it seems incorrect.  The `buf` pointer is incremented during the loop, so it is wrong to say that `LONG*3` bytes from the *current* value of `buf` are being accessed.
```suggestion
                    : "r"(buf), "m"(*buf));
```